### ISSUE TITLE
Images not found

### ISSUE THREAD

**Issue by Sany-IO** [KiweeEu/shopware-kubernetes-dev#4]
Hello,

I work currently on a kubernetes cluster, with arm64 nodes, and found this.
The Problem is, on docker.io the images are not online/found: kiweeteam/shopware-demo-js:6.4.11.1

**Comment by tom00** [comment]
Yes, unfortunately the project is outdated, however we're already revising it to make it compilant with the latest versions of Shopware, Kubernetes and DevSpace.

**Comment by Sany-IO** [comment]
thanks tom, can you inform me? :)
